service/controller/chart/v1/resource/release/update.go

```diff
@@ -97,7 +97,12 @@ func (r *Resource) newUpdateChange(ctx context.Context, obj, currentState, desir
 		return nil, nil
 	}
 
-	isModified := !isEmpty(currentReleaseState) && isReleaseModified(currentReleaseState, desiredReleaseState)
+	isReleaseModified, err := isReleaseModified(currentReleaseState, desiredReleaseState)
+	if err != nil {
+		return nil, nil
```

Comment:
Is it already in the discussion? not return the error in this case?

Reply:
Thanks for checking. Only the desired state is affected. I'll describe the problem when I create the TODO issue.